src/FieldTrait.php

```
@@ -20,6 +20,44 @@ trait FieldTrait {
 
+  /**
+   * Assert that field is empty.
+   *
+   * @code
+   * Then the field "Name" is empty
+   * @endcode
+   *
+   * @Then the field :field should be empty
+   */
+  public function fieldAssertEmpty(string $field): void {
+    $field_element = $this->fieldAssertExists($field);
```

PHPMD (2.15.0)

The variable $field_element is not named in camelCase. (undefined)

(CamelCaseVariableName)